.github/workflows/x-keycloak-operator.yml

```diff
@@ -146,3 +146,87 @@ jobs:
       - name: Show Container tags
         run: |
           echo "${{ steps.meta.outputs.tags }}" >> $GITHUB_STEP_SUMMARY
+
+      - name: Install Yq
+        if: ${{ inputs.tag != 'nightly' }}
+        working-directory: keycloak
+        run: sudo snap install yq
+
+      - name: Install OLM
+        if: ${{ inputs.tag != 'nightly' }}
+        working-directory: keycloak
+        run: ./operator/scripts/install-olm.sh
+
+      - name: Fetch previous version of Keycloak
+        if: ${{ inputs.tag != 'nightly' }}
+        working-directory: keycloak
+        # Could be used version of the last tag from the keycloak-rel/community-operators repo
+        run: echo ::set-output name=previous_tag::$(git describe --tags $(git rev-list --tags --max-count=1))
+
+      - name: Create OLM Bundle
+        if: ${{ inputs.tag != 'nightly' }}
+        working-directory: keycloak
+        run: ./operator/scripts/create-olm-bundle.sh ${{ inputs.tag }} ${{ steps.vars.outputs.previous_tag }} quay.io/${{ inputs.quay-org }}/keycloak-operator
+
+      - name: Clone community-operators
+        if: ${{ inputs.tag != 'nightly' }}
+        uses: actions/checkout@v3
+        with:
+          # keycloak-rel/community-operators
+          repository: keycloak-rel-testing/community-operators
```

Comment:
Should probably be a fork in the keycloak-bot user, and not in keycloak-rel-testing.

Reply:
+1